__tests__/fixtures/requirements/requirements.txt

```diff
@@ -0,0 +1 @@
+ruff==0.8.3
```

Comment:
This file does not need an own folder. It can be moved to `__tests__/fixtures/requirements.txt`